linear_search/Python/linear_search.py

```diff
@@ -0,0 +1,10 @@
+def linear_search(values, search_for):
+    search_at = 0
+    search_res = False
+    while search_at < len(values) and search_res is False:
+        if values[search_at] == search_for:
+            search_res = True
+        else:
+            search_at = search_at + 1
+
+    return search_res
```

Comment:
W292 no newline at end of file

*Origin: PycodestyleBear (W292), Section: `all.autopep8`.*